src/sdf.cpp

```diff
@@ -119,11 +119,9 @@ vec3 Position(ivec4 gridIndex, vec3 origin, vec3 spacing) {
 
 double BoundedSDF(ivec4 gridIndex, vec3 origin, vec3 spacing, ivec3 gridSize,
                   double level, std::function<double(vec3)> sdf) {
-  auto Min = [](ivec3 p) { return std::min(p.x, std::min(p.y, p.z)); };
-
   const ivec3 xyz(gridIndex);
-  const int lowerBoundDist = Min(xyz);
-  const int upperBoundDist = Min(gridSize - xyz);
+  const int lowerBoundDist = minelem(xyz);
```

Comment:
The naming of this function is funny, I was thinking about "lem" as lemma and thinking about what "mine lemma" means :P

Anyway, it is nice that this is already provided by linalg.